src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -78,4 +80,42 @@ public function storeAllowsPaying($paymentMethodName)
 
         $this->paymentMethodRepository->add($paymentMethod);
     }
+
+    /**
+     * @Given the store has a payment method :paymentMethodName with a code :paymentMethodCode
+     */
+    public function theStoreHasAPaymentMethodWithACode($paymentMethodName, $paymentMethodCode)
+    {
+        $paymentMethod = $this->createPaymentMethodFromNameAndCode($paymentMethodName, $paymentMethodCode);
+
+        $this->sharedStorage->set('payment_method', $paymentMethod);
+        $this->paymentMethodRepository->add($paymentMethod);
+    }
+
+    /**
+     * @Given the store has a payment method :paymentMethod disabled
+     */
+    public function theStoreHasAPaymentMethodDisabled(PaymentMethodInterface $paymentMethod)
+    {
+        $paymentMethod->disable();
+
+        $this->paymentMethodRepository->add($paymentMethod);
```

Comment:
Simple `$this->manager->flush()` will be enough here since the entity is already added (persisted).